docsite/source/hash-schemas.html.md

```diff
@@ -124,6 +124,29 @@ user_hash['name' => 'Jane', 'city' => 'London']
 # => Dry::Types::UnknownKeysError: unexpected keys [:city] in Hash input
 ```
 
+### Merging schemas
+
+Similar than inheritance, two schemas can be merged.
+The resulting schema will have the sum of both sets
+of attributes.
+
+user_hash = Types::Hash.schema(
+  name: Types::String
+)
+
+address_hash = Types::Hash.schema(
+  address: Types::String
+)
+
+user_with_address_schema = user_hash.merge(address_hash)
+
+user_with_address_schema['name' => 'Jane', 'address' => 'C/ Foo']
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/gh/dry-rb/dry-types/pullRequest?prid=4765184)